packages/node_modules/@webex/internal-plugin-ediscovery/src/index.js

```diff
@@ -90,58 +90,7 @@ registerInternalPlugin('ediscovery', EDiscovery, {
         name: 'decryptReportRequest',
         direction: 'inbound',
         fn(ctx, object) {
```

Comment:
All of the transforms below have been relocated to a new class as static methods for testability